README.md

```diff
@@ -9,8 +9,9 @@ Its main purpose is to check that the pipeline infrastructure is working correct
 To run the example, from the `pipelines_check` root directory do:
 
 ```
-$ source $LSST_HOME/loadLSST.sh
-$ setup -r .
+$ source $LSST_HOME/loadLSST.bash
+$ setup lsst_distrib
+$ setup -j -r .
```

Comment:
This suggests that there's something missing from the table file.  You should never need to setup something else before `-r .`.

Reply:
I just tried it locally and `setup -t bNNNN -r .` works fine for me and I'm pretty sure that I've asked Jenkins to build just `pipelines_check` before without building anything else. `pipelines_check` depends only on things that are already in lsst_distrib. @yalsayyad what error are you seeing?

Reply:
```
[yusra@lsst-devl01 ~]$ source /project/yusra/LVV-T362/lsst_stack/loadLSST.bash 
(lsst-scipipe-0.7.0) [yusra@lsst-devl01 ~]$ cd /project/yusra/LVV-T362/lsst_stack/pipelines_check
(lsst-scipipe-0.7.0) [yusra@lsst-devl01 pipelines_check]$ setup -r . 
setup: in file /project/yusra/LVV-T362/lsst_stack/pipelines_check/ups/pipelines_check.table: Product pipe_tasks not found
```

Reply:
If you specify a tag does it work? `setup -t TAG -r .`? I have no idea how `current` interacts with this but maybe if EUPS doesn't have a `current` tag for `pipelines_check` it can't work out what tag to use for the dependencies?

Reply:
```(lsst-scipipe-0.7.0) [yusra@lsst-devl01 pipelines_check]$ eups list pipe_tasks
   23.0.0+f2fbba1123    v23_0_0
   git        
(lsst-scipipe-0.7.0) [yusra@lsst-devl01 pipelines_check]$ eups list lsst_distrib
   23.0.0+e2b4167800    current v23_0_0
(lsst-scipipe-0.7.0) [yusra@lsst-devl01 pipelines_check]$ setup -t current -r . # no
setup: in file /project/yusra/LVV-T362/lsst_stack/pipelines_check/ups/pipelines_check.table: Product pipe_tasks not found
(lsst-scipipe-0.7.0) [yusra@lsst-devl01 pipelines_check]$ setup -t v23_0_0 -r . # yes
(lsst-scipipe-0.7.0) [yusra@lsst-devl01 pipelines_check]$ 
```

Reply:
It's really strange that there are only a few packages (`pipe_tasks` being one) that do not have a `current` tag:
```
0a1
> afw
6a8
> base
7a10
> coadd_utils
11a15
> ctrl_execute
15a20
> daf_base
16a22
> daf_persistence
26a33,34
> ip_diffim
> ip_isr
33a42,45
> meas_algorithms
> meas_astrom
> meas_base
> meas_deblender
51a64
> obs_test
52a66,68
> pex_config
> pex_exceptions
> pipe_base
53a70
> pipe_tasks
59a77,78
> shapelet
> skymap
60a80
> utils
```

Reply:
Manually running pipelines check again for verifying 25.0.0, and we're in the same situation. instructions say `setup -r .` but no current tags for everything:
```
(lsst-scipipe-5.0.1) [yusra@sdfrome001 pipelines_check]$ setup -r . 
setup: in file /sdf/data/rubin/user/yusra/DM-32561/lsst_stack/pipelines_check/ups/pipelines_check.table: Product pipe_tasks not found
```

Reply:
Did you mean to not specify the `-j` in your `setup` call? If lsst_distrib is already set up then pipelines_check should set up fine. There is nothing in the pipelines_check table file that isn't in lsst_distrib already.

Reply:
> Did you mean to not specify the -j in your setup call? 

Adding the ` -j` is the whole purpose of this pull request.